.github/workflows/release.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           java-version: 17
 
       - name: Build and publish artifacts
-        run: ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository -Pfinal=true
+        run: ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository -Pfinal=true --no-build-cache --no-configuration-cache --no-parallel
```

Comment:
Thanks! Unfortunately, I think this is the safest way to get a consistent build output.